ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/PartyManagementServiceIT.scala

```diff
@@ -30,8 +31,7 @@ final class PartyManagementServiceIT
     extends LedgerTestSuite
     with PartyManagementItUtils
```

Comment:
As discussed offline: what would prevent us to use different test classes as opposed to squeezing all the tests with the self type constrained test traits? I would argue that would make inheritance much simpler (inheriting from a base, as opposed to inherit from multiple trait with self types...)

Reply:
Are you proposing to have "PartyManagementServiceIT" and "PartyManagementServiceUpdateRpcTestsIT" and "PartyManagementObjectMetaTests"?
This would impact the usability of the test tool from the cli because now you can do e.g. "--include=PartyManagementServiceIT" and have all tests for Party Management included. I think it's a nice ux property.

Reply:
Can it be that this nice property was more important as you were developing, than in general? Also I think with subsequent --include params you can simulate the same effect.
I would prefer to keep inheritance streamlined.

Reply:
Ok, streamlined